File: LanguAI.Frontend/src/api/models/save-learning-request-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { LanguageLevelEnum } from '../models/language-level-enum';
export interface SaveLearningRequestViewModel {
  id?: number | null;
  isActive?: boolean;
  languageId?: number;
  languageLevel?: LanguageLevelEnum;
  userId?: number;
}

File: LanguAI.Backend/LanguAI.Backend/ViewModels/Learning/SaveLearningRequestViewModel.cs
using LanguAI.Backend.Core.Enums;

namespace LanguAI.Backend.ViewModels.Learning;

public class SaveLearningRequestViewModel
{
    public int? Id { get; set; }

    public LanguageLevelEnum LanguageLevel { get; set; }

    public int UserId { get; set; }

    public int LanguageId { get; set; }

    public bool IsActive { get; set; } = true;
}
